src/components/InfoTooltip/InfoTooltip.tsx

```diff
@@ -1,11 +1,9 @@
 import React, { FC } from 'react';
 import './InfoTooltip.scss';
-import { Tooltip } from '../Tooltip';
+import { Tooltip, TooltipProps } from '../Tooltip';
 import { IconQuestionMark } from '../Icons';
 
-export type InfoTooltipProps = {
-  content: string;
-};
+export type InfoTooltipProps = TooltipProps;
```

Comment:
nit: could we do something like `export type InfoTooltipProps extends TooltipProps {}` instead?